File: epoxy-sample/src/main/java/com/airbnb/epoxy/sample/SampleApplication.java
package com.airbnb.epoxy.sample;

import android.app.Application;

import com.facebook.soloader.SoLoader;

public class SampleApplication extends Application {
  @Override
  public void onCreate() {
    super.onCreate();
    // This is for the optional Litho integration
    SoLoader.init(this, false);
  }
}

File: epoxy-composeinterop-maverickssample/src/main/java/com/airbnb/epoxy/composeinterop/maverickssample/SampleApplication.kt
package com.airbnb.epoxy.composeinterop.maverickssample

import android.app.Application
import com.airbnb.mvrx.Mavericks

class SampleApplication : Application() {
    override fun onCreate() {
        super.onCreate()
        Mavericks.initialize(this)
    }
}
